powershell-gallery/powershellget-3.x/Microsoft.PowerShell.PSResourceGet/Publish-PSResource.md

```diff
@@ -16,17 +16,18 @@ Publishes a specified module from the local computer to PSResource repository.
 ## SYNTAX
 
 ```
-Publish-PSResource [-ApiKey <String>] [-Repository <String>] [-Path] <String>
- [-DestinationPath <String>] [-Credential <PSCredential>] [-SkipDependenciesCheck]
- [-SkipModuleManifestValidate] [-Proxy <Uri>] [-ProxyCredential <PSCredential>] [-WhatIf] [-Confirm]
+Publish-PSResource [-NupkgPath <String>] [-ApiKey <String>] [-Repository <String>]
+ [-Path] <String> [-DestinationPath <String>] [-Credential <PSCredential>]
+ [-SkipDependenciesCheck] [-SkipModuleManifestValidate] [-Proxy <Uri>]
+ [-ProxyCredential <PSCredential>] [-WhatIf] [-Confirm]
```

Comment:
You added the `-NupkgPath` parameter but didn't document it in the `## PARAMETERS` section.

Also, this needs to be coordinated with the release of the version of the module that contains this change.